tests/test_api.py

```diff
@@ -56,21 +56,19 @@ def upload_file(self, bundle_file):
                 source_url=bundle_file.url
             )
         )
-        response_data = json.loads(response[1])
-        self.assertIs(type(response_data), dict)
-        self.assertIn('version', response_data)
-        return response_data['version']
+        self.assertIs(type(response_json), dict)
```

Comment:
The text of the response is response[1].
The text of the response parsed into python structures from JSON is response[2].

I removed the call to `json.loads()`, so I just use response[2] directly.